### PR TITLE
adding a test case for opengrok python scripts

### PR DESCRIPTION
**Pull request by tulinkry**
Quick sanity check before deployment.

### CODE REVIEW

File: opengrok-tools/src/test/python/test_binaries.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import unittest
+
+from parameterized import parameterized
+
+from opengrok_tools.utils.command import Command
+
+
+class TestApp(unittest.TestCase):
+    @parameterized.expand((
+            ('opengrok'),
```
Comment (vladak) [inline opengrok-tools/src/test/python/test_binaries.py]: ?

File: opengrok-tools/src/test/python/test_binaries.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    def test_opengrok_binary(self, command):
+        """
+        Test that installed command is able to run
+        :param command: the command name
+        :return:
+        """
+        pass
+        cmd = Command([command, '--help'])
+        cmd.execute()
+        self.assertEqual(0, cmd.getretcode())
```
Comment (vladak) [inline opengrok-tools/src/test/python/test_binaries.py]: what happens if the program ends with exception ?

Comment (tulinkry) [reply opengrok-tools/src/test/python/test_binaries.py]: When I fix the PATH issue the test fails

```
Traceback (most recent call last):
  File "/Users/ktulinger/OpenGrok/opengrok-tools/target/.eggs/parameterized-0.6.1-py3.6.egg/parameterized/parameterized.py", line 392, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/Users/ktulinger/OpenGrok/opengrok-tools/target/src/test/python/test_binaries.py", line 55, in test_opengrok_binary
    self.assertEqual(0, cmd.getretcode())
AssertionError: 0 != 1
```

**Comment by tulinkry** [comment]
Again it’s visible in few builds ago that the appveyor is successful

**Pull request merged**